cockatrice/src/tab_supervisor.cpp

```diff
@@ -621,11 +621,17 @@ void TabSupervisor::updateCurrent(int index)
         emit setMenu();
 }
 
+/**
+ * Determine if a user is a moderator/administrator
+ * By seeing if they have the admin or log tabs open
+ * @return if one of the two tabs are open
+ */
 bool TabSupervisor::getAdminLocked() const
 {
-    if (!tabAdmin)
+    if (!tabAdmin && !tabLog)
```

Comment:
this segfaults if tabadmin is uninitialized while tabLog is true

Reply:
True. I'm kinda planning ahead for the other PR with this change, so I'll clean it up